content/fr/elao/symfonycon-2015.md

```diff
@@ -0,0 +1,251 @@
+slug:               "symfonycon-2015"
+description:        "Nous étions à la SymfonyCon 2015 pour fêter les 10 ans de Symfony"
+language:           "fr"
+thumbnail:          "/images/posts/thumbnails/symfonycon-2015.jpg"
+header_img:         "/images/posts/headers/foliesbergeres.jpg"
+tags:               ["Conférence", "Symfony", "SymfonyCon"]
+categories:         ["conference"]
+
+author_username:    "rhanna"
+---
+
+## Symfony a 10 ans
+
+Symfony a fêté ses 10 ans pendant deux jours dans un lieu très spécial, les Folies Bergères à Paris. Et nous avions la
+joie d'y être !
+
+### Keynote d'ouverture
+
+La keynote d'ouverture animée par Fabien Potencier a mis en lumière les visages des acteurs clés ayant contribué à la
+réussite de ce framework : les dévelopeurs, les ambassadeurs et les contributeurs à sa documentation.
+
+À retenir :
+
+* Le passage de Symfony de la version 1 à la 2 a pris 4 ans. De la version 2 à la 3, également 4 ans. Symfony 4 est
+prévue plus tôt, dans deux ans, soit en 2017;
+* La certification Symfony précédemment liée à la version 2.3 est maintenant disponible pour la version 3.0;
+* L'un des chantiers pour le futur de Symfony est de rendre le *filesystem*, *read-only*, ce que signifie que la
+gestion du cache devra être revue.
+
+### Dig in Security with Symfony by Sarah Khalil
+
+Plongeon au coeur du composant *Security* de Symfony. Ce qu'il faut retenir c'est qu'il ne faut plus utiliser les
+[ACL](http://symfony.com/doc/current/cookbook/security/acl.html)
+mais les [Voters](http://symfony.com/doc/current/cookbook/security/voters.html).
+Rien de nouveau donc, dommage que cela soit une redite de la documentation.
+
+[Les slides](https://speakerdeck.com/saro0h/symfonycon-paris-dig-in-security)
+
+### The cloud is the future, and your architecture isn't ready by David Zuelke
+
+Retour d'expérience d'Heroku sur les bonnes pratiques pour rendre une application éligible au Cloud.
+Ne pas mettre la configuration de vos environnments (dev, prod...) dans votre code mais directement sur la plateforme.
+Utiliser  *getenv($var)* dans le code pour récupérer une variable d'environnement.
+Pour en savoir plus sur l'ensemble de ces bonne pratiques, un site a été créé : [12factor](http://12factor.net/).
+
+### Building high profile webapps with Symfony and API Platform by Kévin Dunglas
+
+Retour d'expérience sur un projet pour BeinSports : l'implémentation d'une plateforme API et de multiples consommateurs
+de cette API de nature différente : backoffice en Symfony, site web, applications mobiles, Xbox.
+
+Ce qu'il faut retenir en vrac :
+
+* Toute la logique métier est gérée par l'API.
+* Les consommateurs de l'API ne gèrent que la présentation des informations.
+* La plateforme API est sous Symfony ; les consommateurs côté *frontend* ne génèrent pas de routes en PHP ou Symfony,
+ils se calquent sur l'URI de l'API.
+
+J'ai beaucoup aimé ce retour d'expérience même si c'est dommage que la présentation ait été simplement lue et que cela
```

Comment:
même si c'est dommage __que__ la présentation ait été simplement lue et __que__ cela

Un peu lourd. Quelque chose de plus léger comme 

`J'ai beaucoup aimé ce retour d'expérience même si l'on peut regretter que le manque de spontanéité de la  présentation`